asset/src/job_metric_api/exporter.ts

```diff
@@ -0,0 +1,24 @@
+import promClient from 'prom-client';
+import { JobMetricAPIConfig } from './interfaces';
+
+const express = require('express');
+
+export default async function createExporter(
+    jobMetricApiConfig: JobMetricAPIConfig
+): Promise<void> {
+    if (jobMetricApiConfig.default_metrics) {
+        promClient.collectDefaultMetrics();
+    }
+
+    const metricServer = express();
+
+    metricServer.get('/', (_req: any, res: { send: (arg0: string) => void; }) => {
```

Comment:
Use the types from `express` to for request and response, this will help detect errors

```suggestion
    metricServer.get('/', (_req: Request, res: Response) => {
```